react-stopwatch-component/src/stopwatch.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+
+export default class Stopwatch extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { count: 0, isCounting: false };
+    this.clearCounter = this.clearCounter.bind(this);
+    this.startCounting = this.startCounting.bind(this);
+    this.resetCounting = this.resetCounting.bind(this);
+
+  }
+
+  startCounting() {
+    this.setState({ isCounting: true });
+    this.interval = setInterval(() => this.setState(state => ({
```

Comment:
I would define this in the constructor function initially, and then update it as needed.
for example this.interval = 0;